flang/test/Lower/OpenMP/omp-parallel-allocate-clause.f90

```diff
@@ -0,0 +1,29 @@
+! This test checks lowering of OpenMP parallel Directive with
+! `PRIVATE` clause present.
+
+! RUN: %bbc -fopenmp -emit-fir %s -o  - | \
+! RUN:   FileCheck %s --check-prefix=FIRDialect
+
+!FIRDialect: func @_QPallocate_clause(%arg0: !fir.ref<f32>, %arg1: !fir.ref<f32>) {
+!FIRDialect-DAG: %0 = fir.alloca i32 {name = "_QFallocate_clauseEa"}
+!FIRDialect-DAG: %1 = fir.alloca i32 {name = "_QFallocate_clauseEalpha"}
+!FIRDialect-DAG: %2 = fir.alloca i32 {name = "_QFallocate_clauseEb"}
+!FIRDialect-DAG: %3 = fir.alloca i32 {name = "_QFallocate_clauseEbeta"}
+!FIRDialect-DAG: %4 = fir.alloca i32 {name = "_QFallocate_clauseEgama"}
+!FIRDialect-DAG: %5 = fir.alloca i32 {name = "_QFallocate_clauseEx"}
+!FIRDialect-DAG: %6 = fir.alloca i32 {name = "_QFallocate_clauseEy"}
+!FIRDialect-DAG: %7 = fir.alloca i32 {name = "_QFallocate_clauseEz"}
+  
+!FIRDialect-DAG: omp.parallel private(%1 : !fir.ref<i32>, %3 : !fir.ref<i32>, %4 : !fir.ref<i32>) allocate(%c1_i32 : i32 -> %5 : !fir.ref<i32>, %c1_i32 : i32 -> %6 : !fir.ref<i32>, %c1_i32 : i32 -> %7 : !fir.ref<i32>, %c1_i32_0 : i32 -> %0 : !fir.ref<i32>, %c1_i32_0 : i32 -> %2 : !fir.ref<i32>) {
+!FIRDialect:    omp.terminator
+!FIRDialect:  }
+subroutine allocate_clause(arg1, arg2)
+use omp_lib
```

Comment:
Is this line strictly needed ? I reckon you can do this without using `use omp_lib` and get away with the change introduced in `flang/test/lit.cfg.py`

Reply:
I'll try this and see if the test still passes.

Reply:
Since a predefined value (omp_large_cap_mem_alloc) is used, using the omp_lib is the right way.